internal/runner/testdata/rspec/Gemfile

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
```

Comment:
I added thi so we can run `bktec` from this project using the default command `bundle exec rspec ...`